style_editor.html

```diff
@@ -8,7 +8,7 @@
   <title>ProffieOS Style Editor</title>
 <style>
 
-/* Make page elements non-selectable with drag box */
+/* Make page elements like buttons and whitespace non-selectable with drag box */
```

Comment:
This PR has WAY too much stuff in it.
Can we create a settings dialog with just *one* setting in it first?
Also, all the "minor cleanup" stuff is getting in the way of reviewing the settings code.